app/client/src/components/AnimatedGridLayout/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+/** Regex used to strip everything but numbers from measurement units. */
+export const NON_DIGIT_REGEX = /\D+/gm;
```

Comment:
Found out this breaks if the number has a decimal point. 
Try this one to fix it

```suggestion
export const NON_DIGIT_REGEX = /(\d+(\.\d+)?)/gm;
```

Reply:
Slightly modified original regex to fix issue with decimal point.